src/client.ts

```diff
@@ -145,6 +145,25 @@ export class APIClient {
       },
     );
   }
+
+  /**
+   * Iterates each domain resource in the provider.
+   *
+   * @param iteratee receives each resource to produce entities/relationships
+   */
+  public async iterateDomains(
+    install_id: string,
```

Comment:
Very minor thing but we usually use camelCase notation whenever possible (some example where it may not be possible is for types where we want to match the provider's returned objects - as it's the case in your WpEngineInstall for example and others).